packages/components/Button/src/Button.styling.ts

```diff
@@ -58,6 +58,7 @@ export const stylingSettings: UseStylingOptions<ButtonProps, ButtonSlotProps, Bu
       (tokens: ButtonTokens, theme: Theme) => ({
         style: {
           display: 'flex',
+          overflow: 'hidden',
```

Comment:
just for my own understanding -- if someone did want to revert this behavior, they still can using custom tokens, right?

Reply:
Yeah, I think so. Or adding their own stiling on top via our slot stuff